rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/message/RocketChatAvatar.java

```diff
@@ -0,0 +1,66 @@
+package chat.rocket.android.widget.message;
```

Comment:
This class is not only for message. So it would be better to put this class in `chat.rocket.android.widget`